### DISCUSSION TITLE
已付款无法收到激活码邮件

### DISCUSSION THREAD

**Discussion by stardu0707** [getlantern/lantern-forum-cn#765] [General]
是否需要提供支付截图等信息，请关注感谢

**Comment by stardu0707** [comment]
![5df93c35-e327-4c75-9a1a-a99b3c48ecd2](https://github.com/user-attachments/assets/c4fe5f35-1356-42e0-adff-fc7b3bf3b894)

**Comment by stardu0707** [comment]
邮箱为357857175@qq.com

**Comment by MaxMKMax** [comment]
@stardu0707 您好. 谢谢您提供的信息. 我会帮您提交问题的. 谢谢. 请您过几天留意这个帖子或您的邮箱

**Reply by stardu0707** [reply]
非常感谢帮忙提交问题反馈，可否再帮忙关注下进展如何，已经有十天了，还没收到信息

**Comment by stardu0707** [comment]
非常感谢帮忙提交问题反馈，可否再帮忙关注下进展如何，已经有十天了，还没收到信息



---原始邮件---
发件人: "Mike ***@***.***&gt;
发送时间: 2024年8月8日(周四) 晚上11:46
收件人: ***@***.***&gt;;
抄送: ***@***.******@***.***&gt;;
主题: Re: [getlantern/lantern-forum-cn] 已付款无法收到激活码邮件 (Discussion #765)




 
@stardu0707 您好. 谢谢您提供的信息. 我会帮您提交问题的. 谢谢. 请您过几天留意这个帖子或您的邮箱
